Shared/Samples/Identify layer features/IdentifyLayerFeaturesView.swift

```diff
@@ -0,0 +1,141 @@
+    }
+    
+    var body: some View {
+        ZStack {
+            MapViewReader { proxy in
+                MapView(map: map)
+                    .onSingleTapGesture { screenPoint, _ in
+                        tapScreenPoint = screenPoint
+                    }
+                    .task {
+                        do {
+                            // Add map image layer to the map.
+                            let mapImageLayer = ArcGISMapImageLayer(url: .worldCities)
+                            try await mapImageLayer.load()
+                            
+                            // Hide continent and world layers.
+                            mapImageLayer.subLayerContents[1].isVisible = false
+                            mapImageLayer.subLayerContents[2].isVisible = false
+                            map.addOperationalLayer(mapImageLayer)
+                            
+                            // Add feature layer to the map.
+                            let featureTable = ServiceFeatureTable(url: .damageAssessment)
+                            try await featureTable.load()
+                            let featureLayer = FeatureLayer(featureTable: featureTable)
+                            map.addOperationalLayer(featureLayer)
+                        } catch {
+                            // Present error load the layers if any.
+                            self.error = error
+                        }
+                    }
+                    .task(id: tapScreenPoint) {
+                        // Identify layers using the screen point.
+                        if let screenPoint = tapScreenPoint,
+                           let results = try? await proxy.identifyLayers(
+                            screenPoint: screenPoint,
+                            tolerance: 12,
+                            returnPopupsOnly: false,
+                            maximumResultsPerLayer: 10
+                           ) {
+                            handleIdentifyResults(results)
+                        }
+                    }
+                    .overlay(alignment: .top) {
+                        Text(overlayText)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(8)
+                            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    }
+                    .alert(isPresented: $isShowingAlert, presentingError: error)
+            }
+        }
+    }
+}
+
+private extension IdentifyLayerFeaturesView {
+    /// Updates the overlay text based on the identify layer results.
+    /// - Parameter results: An `IdentifyLayerResult` array to handle.
+    func handleIdentifyResults(_ results: [IdentifyLayerResult]) {
+        // Get layer names and geoelement counts from the results.
+        let layerNameElementCount: [(layerName: String, geoElementsCount: Int)] = results.map { identifyLayerResult in
```

Comment:
Is `identifyLayerResultInfo` a better name for this variable?

Reply:
The suggestion was to use `identifyLayerResultInfo` instead of `layerNameElementCount` and not for function name.